.babelrc

```diff
@@ -0,0 +1,4 @@
+{
+  "presets": ["react", "es2015", "es2016", "es2017", "stage-2"],
```

Comment:
ok that does look quite nice. I'm a bit worried about the warning at http://babeljs.io/docs/plugins/ though:

> These proposals are subject to change so use with extreme caution, especially for anything pre stage-3.

Maybe we should be picking out the individual transform rather than using the stage-2 preset.